Source/FolioReaderPage.swift

```diff
@@ -478,7 +482,10 @@ extension UIWebView {
     }
     
     public override func canPerformAction(action: Selector, withSender sender: AnyObject?) -> Bool {
-        
+		guard readerConfig.useReaderMenuController else {
+			return false
```

Comment:
Ok. But then there has to be a way to customize the actions which the `FolioReaderWebView` (aka wich of the default `UIWebView` actions) should perform. Therefore I could make the `FolioReaderWebView` class public. I could then make an extension for it in the app I'm working on or we provide a delegate method. What do you think?